ntbs-service/Pages/Search/Index.cshtml

```diff
@@ -1,54 +1,82 @@
 @page
 @model ntbs_service.Pages_Search.IndexModel
+@using static NHSUK.FrontEndLibrary.TagHelpers.FormGroupType
 @using Microsoft.AspNetCore.Routing
 
 @{
     ViewData["Title"] = "Search";
+    Layout = "../Shared/_NoForm_Layout.cshtml";
 }
 
 <h1>Search</h1>
-
-<h4>Parameters</h4>
-<hr />
 <div>
-    <form method="post">
+    <form method="get">
         <div asp-validation-summary="ModelOnly"></div>
-        <div>
+        <div class="enter-search-details-tip">
+            To search enter at least one of the following fields
         </div>
-        <div>
-            <input type="submit" value="Search" />
+        <nhs-grid-row>
+            <nhs-grid-column grid-column-width="OneQuarter">
+                @{
+                    var IdFilterError = !Model.IsValid("IdFilter");
+                    var IdFilterGroupState = IdFilterError ? Error : Standard;
+                }
+                <nhs-form-group nhs-form-group-type=@IdFilterGroupState v-bind:class="{ 'nhsuk-form-group--error': hasError}">

```

Comment:
`v-bind:class="{ 'nhsuk-form-group--error': hasError}"` doesn't work after you submit the the form. As the error doesn't go away even if you enter correct data (Js doesn't clear it properly). Could you please check that this is not the case here?

And do we need this since there is no vue component talking to this bit of code?

Reply:
Ah yep, we don't need any dynamic validation on search so scrapped the vue component